shared/chat/conversation/input-area/normal/platform-input.native.js

```diff
@@ -166,6 +205,11 @@ const Action = ({hasText, onSubmit, isEditing, openFilePicker, insertMentionMark
     </Box>
   ) : (
     <Box2 direction="horizontal" gap="small" style={styles.actionIconsContainer}>
+      <ExplodingIcon
+        explodingModeSeconds={explodingModeSeconds}
+        isExploding={isExploding}
+        openExplodingPicker={openExplodingPicker}
```

Comment:
On native on device this isn't dismissing the keyboard, so there's no way to make a selection without backing out of the convo, going back in, and choosing before focusing the input.

Reply:
Found a solution with a little fiddling, changing `_explodingPickerToggle` to focus or blur the input depending on whether the popup is open works.

Reply:
I've done this within the adapter. Thoughts?

Reply:
Sure, that seems ok.